includes/class-wsu-syndicate-shortcode-people.php

```diff
@@ -508,6 +522,21 @@ private function generate_item_html( $person, $type, $atts ) {
 
 			return $html;
 
+		} elseif ( 'card' === $type ) {
+
+			$heading_tag = ( ! empty( $atts['heading_tag'] ) ) ? $atts['heading_tag'] : 'h3';
```

Comment:
$atts['heading_tag'] will never be empty unless someone specifically tells it to be in the shortcode - but then why wouldn't they just put in h3? Shortcode atts should have the defaults set above.

Reply:
Just remove - not needed.